chairing.py

```diff
@@ -19,43 +17,64 @@
     "Please do not take questions from the room unless and until questions from Slido have been exhausted.",
     "If participants in the room try to get into protracted discussions with the speaker during the Q&amp;A, please encourage them to use the coffee breaks to continue the discussion.",
-    "Please keep an eye on the time during the Q&amp;A and close off questions in sufficient time to allow movement between rooms and setting up for the next speaker.",
+    "Please keep an eye on the time during the Q&amp;A and close off questions in sufficient time to allow movement between rooms and setting up for the next speaker. Remind the volunteer to clear all Slido questions so they don't get mixed with those for the next presentation.",
     "At this point you can pre-announce the next speaker and the start time of their talk.",
 ]
 
 PANEL_SECTION = [
     ("Panels", styles["Heading1"]),
-    "Panels have their own chairs who take on some of the responsibilities above. Panels last 50 minutes, with 10 minutes slack to allow the panel to assemble and get ready at the start, and also to prepare for the next event at the end.",
+    "Your session contains a panel session. Panels have their own chairs who take on some of the responsibilities above. Panels last 50 minutes, with 10 minutes slack to allow the panel to assemble and get ready at the start, and also to prepare for the next event at the end. However, due to scheduling constraints this buffer might not always be possible. Therefore, please ensure any talks before a panel end promptly to allow the panel to get ready.",
     "Once the panelists have assembled and have microphones ready, then please introduce the panel chair and topic similarly as for talks. Once the panel chair takes over and starts to introduce the panelists, please start a timer.",
     "After 45 minutes, please show the “5 minutes remaining” sign to the panel chair. Similarly, at 48, 49, and 50 minutes show the 2, 1, and 0 minutes remaining signs.",
-    "If at 50 minutes the panel shows no signs of wrapping up, please stand up and look conspicuous. If after another minute the discussion or monologue is still continuing and the chair hasn’t taken action, then please interrupt them and ask the chair to wrap things up.",
+    "If at 50 minutes the panel shows no signs of wrapping up, please stand up and look conspicuous. If after another minute the discussion or monologue is still continuing and the chair hasn't taken action, then please interrupt them and ask the chair to wrap things up.",
+]
+
+WALKTHROUGH_SECTION = [
+    ("Walkthroughs", styles["Heading1"]),
+    "Your session includes a Walkthrough. Timings are slightly different than for talks given above, although most of the other guidance holds true. TODO explain timings and confirm with @cc-a that there are no gaps between walkthroughs.",
```

Comment:
@cc-a This will need to be extended for the requirements of the Walkthrough. Please amend timings as needed. Also, can I confirm that you are leaving no buffer period between the walkthroughs?